src/man/kiwix-serve.1

```diff
@@ -1,60 +1,130 @@
 .TH KIWIX 1 "21 May 2012"
+
 .SH NAME
-kiwix\-serve \- Kiwix HTTP Server
+kiwix-serve \- Kiwix HTTP Server
+
 .SH SYNOPSIS
-.IX Header "SYNOPSIS"
-.br
-kiwix\-serve [\-\-index=INDEX_PATH] [\-\-port=PORT] [\-\-verbose] [\-\-daemon] [\-\-attachToProcess=PID] ZIM_PATH
+.B kiwix-serve
+[\fB-h\fR] [\fB-a\fR \fBPID\fR] [\fB-d\fR] [\fB-i\fR \fBADDRESS\fR] [\fB-M\fR]
+[\fB-m\fR] [\fB-n\fR] [\fB-b\fR] [\fB-p\fR \fBPORT\fR] [\fB-r\fR \fBURL_ROOT_LOCATION\fR]
+[\fB-s\fR \fBSEARCH_LIMIT\fR] [\fB-t\fR \fBTHREADS\fR] [\fB-v\fR] [\fB-V\fR]
+[\fB-z\fR] [\fB-c\fR \fBCUSTOM_INDEX\fR] [\fB-L\fR \fBIP_CONNECTION_LIMIT\fR]
+[\fB--library\fR] \fBLIBRARY_PATH\fR
 .br
-kiwix\-serve \-\-library [\-\-port=PORT] [\-\-verbose] [\-\-daemon] [\-\-attachToProcess=PID] LIBRARY_PATH
+.B kiwix-serve
+[\fB-h\fR] [\fB-a\fR \fBPID\fR] [\fB-d\fR] [\fB-i\fR \fBADDRESS\fR] [\fB-M\fR]
+[\fB-m\fR] [\fB-n\fR] [\fB-b\fR] [\fB-p\fR \fBPORT\fR] [\fB-r\fR \fBURL_ROOT_LOCATION\fR]
+[\fB-s\fR \fBSEARCH_LIMIT\fR] [\fB-t\fR \fBTHREADS\fR] [\fB-v\fR] [\fB-V\fR]
+[\fB-z\fR] [\fB-c\fR \fBCUSTOM_INDEX\fR] [\fB-L\fR \fBIP_CONNECTION_LIMIT\fR]
+[\fBZIM_PATH\fR]...
```

Comment:
Why duplicate all options in the two invocation variants of `kiwix-serve`? Isn't the approach used for the [online documentation](https://kiwix-tools.readthedocs.io/en/latest/kiwix-serve.html#usage) better?